style.css

```diff
@@ -1,42 +1,79 @@
-/*
- * Mums in Tech | Week 03
- *
- * I have put some basic CSS in place to give everyone a
- * nice starting point. Feel free to change it however.
- *
- * You challenge, if you choose to accept it, is to
- * put your own design/ideas into the provided HTML with
- * minimal changes to the HTML itself.
- */
-
 body {
-  /* I prefer sans-serif (without curls) so I'm changing it haha */
-  font-family: Helvetica, sans-serif;
+  font-family: "Comic Sans MS", cursive, sans-serif;
```

Comment:
I shudder at your choice of font but correctly used and good to see the inclusion of `cursive` too :+1: